app/src/main/java/fr/free/nrw/commons/upload/UploadModel.java

```diff
@@ -428,6 +431,34 @@ public String getFileName() {
         public Place getPlace() {
             return place;
         }
+
+        public Uri getContentUri() {
```

Comment:
This is not required. Instead of generating content uri here, simply pass it to `UploadItem` from `UploadableFile`.